content/webapp/components/CollaboratorCard/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { FunctionComponent } from 'react';
+
+import styled from 'styled-components';
+import Icon from '@weco/common/views/components/Icon';
+import { IconSvg } from '@weco/common/icons';
+import NextLink from 'next/link';
+
+const StyledLink = styled(NextLink).attrs({ className: 'font-size-6' })`
```

Comment:
We have a `font()` helper function to assign the correct class [here](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/utils/classnames.ts) that you could use.